tox.ini

```diff
@@ -30,7 +31,9 @@ deps =
     pytest==4.6.3
     pytest-cov==2.7.1
     pytest-helpers-namespace==2019.1.8
+    pytest-html>=1.21.1
```

Comment:
I wouldn't add this as we don't need it. You can generate reports yourself locally.

Reply:
https://github.com/ansible/molecule/pull/2144#discussion_r300634637